### PR TITLE
GPC-133: Add cognito client creation for admins

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: src/main/kotlin/uk/gov/gdx/datashare/controllers/AdminController.kt [hunk 1]
```diff
@@ -31,4 +37,25 @@ class AdminController(
...
   fun getEvents() = eventDataRepository.findAll().toList()
+
+  @PostMapping("/cognitoClients")
+  @Operation(
+    summary = "Create cognito client",
+    description = "Need scope of events/admin",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Event",
```
Comment (ethanmills) [inline src/main/kotlin/uk/gov/gdx/datashare/controllers/AdminController.kt:48]: Is this correct?

File: src/main/kotlin/uk/gov/gdx/datashare/services/CognitoService.kt [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import uk.gov.gdx.datashare.models.CognitoClientRequest
+import uk.gov.gdx.datashare.models.CognitoClientResponse
+
+@Service
+class CognitoService(
+  @Value("\${cognito.user-pool-id}") val userPoolId: String,
+  @Value("\${cognito.enabled}") val enabled: Boolean,
+  @Value("\${cognito.acquirer-scope}") val acquirerScope: String,
+  @Value("\${cognito.supplier-scope}") val supplierScope: String,
+  @Value("\${cognito.admin-scope}") val adminScope: String,
```
Comment (ethanmills) [inline src/main/kotlin/uk/gov/gdx/datashare/services/CognitoService.kt:20]: Can we make these nullable and default to null so we don't need to set the in application-dev? Seems weird to need to set dummy values

File: src/main/kotlin/uk/gov/gdx/datashare/controllers/AdminController.kt (hunk 1 above)
Comment (oskarwilliams378) [reply src/main/kotlin/uk/gov/gdx/datashare/controllers/AdminController.kt:48]: Nope, thanks!

File: src/main/kotlin/uk/gov/gdx/datashare/services/CognitoService.kt (hunk 2 above)
Comment (oskarwilliams378) [reply src/main/kotlin/uk/gov/gdx/datashare/services/CognitoService.kt:20]: Sure

**Pull request merged**